### ISSUE TITLE
Startup

### ISSUE THREAD

**Issue by StormDestroyer** [sciguyryan/TheCrackPack#809]
I am having an issue starting up my minecraft with the launcher i just downloaded the launcher and attached my mc account and i want to play with it on but do not know how??????
Please Help!!!!!!!!!!!

**Comment by evknucklehead** [comment]
Calm down. There are a couple things you need to do to play the CrackPack.

First, make sure you're running a 64-bit OS and 64-Bit Java 7. The CrackPack needs these because of limitations in 32-Bit Java.

Second, make sure you have at least 4GB available memory. Again, the CrackPack needs a lot more memory than vanilla Minecraft.

In the launcher, under the settings section, select the Java/Minecraft tab. There should be a number of options here. We're interested in 3 specific ones. Minimum Memory should be set to somewhere around 256MB. Maximum Memory should be at least 2.5GB (2560MB), but try not to set it to more than half your total memory unless you only have 4-6GB. The last option is PermSize. This should be set to 256MB. 

Now that that's set and saved, go to the Packs section of the launcher. Scroll through the list to find The CrackPack. There will be a couple of buttons available. The one we want is the Create New Instance button. Click that, and it will give you the option to name the instance and select a version of the pack. If you intend to play on a server that someone else has set up, find out which version of the pack they are using and which optional mods (we'll be choosing those next) they run with.

The next screen will have two sides. On the left will be the required mods included in the pack. On the right will be a number of optional mods that you can choose to include. Some of them will be selected by default. Again, if you're planning on playing on someone else's server, check to see which of these to select. If you're not planning on playing on a server, you can look through the various mod descriptions and choose what to install. Note that you cannot select both JourneyMap and Zan's Minimap at the same time, as they both serve the same purpose.

Once you have made your selections, the pack will start to download. There will be at least one instance where the launcher will open a browser window to download a mod. This is how certain mods can be included in the pack without being distributed by the launcher. Simply follow the download instructions and save the mod files to the standard download location. Once that's done downloading, go back to the launcher and click on the continue button. 

After a while (exact time depends on your internet connection) the launcher will tell you that the instance is ready. Click onto the Instances section of the launcher and select the instance you just created. From now on, this is where you will go to launch the pack. Click the play button and patiently wait as the pack loads. This will be considerably longer than loading vanilla Minecraft, as there are between 180 and 190 mods that need to load. From there, you're ready to start playing.

There are a couple places where hiccups can occur. Many of them are covered elsewhere in the issues list, some open, some closed.

**Comment by StormDestroyer** [comment]
Thank you very much I was able to get it working!

**Comment by NobleNik** [comment]
Hi i have this problem on console it says this:Error downloading akka-actor_2.11-2.3.3.jar from http://download.nodecdn.net/containers/atl/packs/TheCrackPack/files/core-libs/akka-actor_2.11-2.3.3.jar. Expected hash of 2a3d8946f684f928c48a2dd298c5cd29 but got e46952e017df7f6f3a7e3cc75cc04f30 instead. Copied to FailedDownloads Folder. Cancelling install! So i use a mac and Java 8 how do i fix this, i want to play The Crack Pack

**Comment by evknucklehead** [comment]
This is an issue that is known to occur on occasion where the files don't download properly from the ATLauncher servers. One option is to download the file directly and place it into the launcher's Downloads folder. On OS X, I think this is inside the program's .app folder, so you'll need to right or control click on the program and select Show Contents to see it. Anyone who has used the launcher on OS X can tell you for sure.

Another thing to make sure of is that the system's built-in firewall (and any add-on firewalls you may have installed) is set to not block the addresses at http://www.atlauncher.com/help/whitelist so the launcher can properly locate all the servers it needs.

**Comment by NobleNik** [comment]
thank you very much

**Comment by evknucklehead** [comment]
With the firewall whitelist done, you can try running the download in the launcher again. 

If it fails again, you should be able to download it from the address listed in the error. I tried looking for a download from the original source, but the only versions I could find there were the current version and a version older than the one you need.

**Comment by NobleNik** [comment]
can i use the current version? is it 1.7.10?

**Comment by NobleNik** [comment]
is the current one 1.7.10? and if so please send me the link

**Comment by NobleNik** [comment]
is it this link here? 
http://bob.nodeservers.net/containers/atl/packs/TheCrackPack/files/core-libs/akka-actor_2.11-2.3.3.jar

**Comment by evknucklehead** [comment]
Yes, that link. Or the one in your earlier post, they should give the same result.

**Comment by NobleNik** [comment]
where do i put it? in the jarmods or the mods?

**Comment by evknucklehead** [comment]
Into the launcher's downloads folder. It will take care of placing it where it needs to go.

**Comment by NobleNik** [comment]
K thanks
